p4-16/psa/PSA.mdk

```diff
@@ -392,12 +392,6 @@ through the rest of the parser into the `Ingress`  control block as one
 might normally expect.  This will be left as an option to the user in
 their P4 programs, not required behavior for all P4 programs.
 
-~TBD
-Is this true for RESUB and RECIRC packets, or is it intended that
-there by a way in PSA to specify some collection of fields to be
-preserved with a resub/recirc'ed packet?
-~
-
```

Comment:
Removed this one.  It seems like it should be covered by whatever we come up with for passing metadata with packets, which is a prominent issue in the open issues section.